### PR TITLE
(feat) Add enime.moe

### PR DESCRIPTION
**Pull request by NADESHIKON**
## Acknowledgements
- [ ] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [ ] I linted the code by running `yarn format`
- [ ] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

image

### CODE REVIEW

**Comment by NADESHIKON** [comment]
image

**Comment by RisingSunLight42** [comment]
> image
> 
> (No idea why the image from PR body doesn't show up)

Let a space of one line at the top of the link.

File: websites/E/Enime/presence.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+		document.evaluate(
+			path,
+			document,
+			null,
+			XPathResult.FIRST_ORDERED_NODE_TYPE,
+			null
+		).singleNodeValue;
+
+presence.on("UpdateData", () => {
+	/*UpdateData is always firing, and therefore should be used as your refresh cycle, or `tick`. This is called several times a second where possible.
```
Comment (RisingSunLight42) [inline websites/E/Enime/presence.ts:27]: Useless comment
```suggestion

```

File: websites/E/Enime/presence.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	const currentPath = document.location.pathname;
+
+	switch (currentPath) {
+		case "/":
+			presenceData.details = "Browsing...";
+			presenceData.state = "Browsing recently released";
+			presenceData.largeImageKey = "unknown";
+			presenceData.smallImageKey = "search";
+			presenceData.smallImageText = "Browsing";
+			presenceData.endTimestamp = null;
```
Comment (RisingSunLight42) [inline websites/E/Enime/presence.ts:40]: Useless comment
```suggestion

```

File: websites/E/Enime/presence.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+			document,
+			null,
+			XPathResult.FIRST_ORDERED_NODE_TYPE,
+			null
+		).singleNodeValue;
+
+presence.on("UpdateData", () => {
+	/*UpdateData is always firing, and therefore should be used as your refresh cycle, or `tick`. This is called several times a second where possible.
+  
+      It is recommended to set up another function outside of this event function which will change variable values and do the heavy lifting if you call data from an API.*/
```
Comment (RisingSunLight42) [inline websites/E/Enime/presence.ts:29]: Useless comment
```suggestion

```

File: websites/E/Enime/presence.ts [hunk 4]
```diff
@@ -0,0 +1,85 @@
+const presence = new Presence({
+		//The client ID of the Application created at https://discordapp.com/developers/applications
+		clientId: "1005706063586279445",
+	}),
+	presenceData: PresenceData = {
+		//The large image on the presence. This can be a key of an image uploaded on the Discord Developer Portal - Rich Presence - Art Assets, or a URL to an image
```
Comment (RisingSunLight42) [inline websites/E/Enime/presence.ts:6]: Useless comment
```suggestion

```

File: websites/E/Enime/presence.ts [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+		largeImageKey: "unknown",
+		//The text which is displayed when hovering over the small image
```
Comment (RisingSunLight42) [inline websites/E/Enime/presence.ts:8]: Useless comment
```suggestion

```

File: websites/E/Enime/presence.ts [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+		smallImageText: "enime.moe",
+		//The upper section of the presence text
```
Comment (RisingSunLight42) [inline websites/E/Enime/presence.ts:10]: Useless comment
```suggestion

```

File: websites/E/Enime/presence.ts [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+		details: "Browsing recently released",
+		//The lower section of the presence text
```
Comment (RisingSunLight42) [inline websites/E/Enime/presence.ts:12]: Useless comment
```suggestion

```

File: websites/E/Enime/presence.ts [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+		state: "Watching...",
+		//The unix epoch timestamp for when to start counting from
```
Comment (RisingSunLight42) [inline websites/E/Enime/presence.ts:14]: Useless comment
```suggestion

```

File: websites/E/Enime/presence.ts [hunk 9]
```diff
@@ -0,0 +1,85 @@
+const presence = new Presence({
+		//The client ID of the Application created at https://discordapp.com/developers/applications
```
Comment (RisingSunLight42) [inline websites/E/Enime/presence.ts:2]: Useless comment
```suggestion

```

File: websites/E/Enime/dist/metadata.json [hunk 10]
```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.7",
+    "author": {
+        "name": "riimuru",
+        "id": "775082234507427890"
+    },
+    "contributors": [{
+        "name": "riimuru",
+        "id": "775082234507427890"
+    }],
```
Comment (RisingSunLight42) [inline websites/E/Enime/dist/metadata.json:10]: Useless since you're the author
```suggestion
```

**Review by RisingSunLight42 — CHANGES_REQUESTED** [review]
Please apply the changes

**Comment by EncryptedDev** [comment]
Low quality PR. Address the coments above and add more functionality before reopening

**Pull request closed without merging**